specificatie/verblijfplaats.yaml

```diff
@@ -0,0 +1,392 @@
+          huisletter:
+            $ref: '#/components/schemas/Huisletter'
+          huisnummertoevoeging:
+            $ref: '#/components/schemas/Huisnummertoevoeging'
+          aanduidingBijHuisnummer:
+            $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+          postcode:
+            $ref: '#/components/schemas/Postcode'
+          adresseerbaarObjectIdentificatie:
+            $ref: '#/components/schemas/AdresseerbaarObjectIdentificatie'
+          nummeraanduidingIdentificatie:
+            $ref: '#/components/schemas/NummeraanduidingIdentificatie'
+          adresregel1:
+            $ref: '#/components/schemas/Adresregel1'
+          adresregel2:
+            $ref: '#/components/schemas/Adresregel2'
+          woonplaats:
+            $ref: '#/components/schemas/Woonplaats'
+          indicatieVestigingVanuitBuitenland:
+            type: boolean
+            description: |
+                          Geeft aan dat de persoon zich vanuit het buitenland heeft ingeschreven.
+          korteNaam:
+            description: |
+                          De officiële openbareruimtenaam uit de Basisregistratie Gebouwen en Adressen (BAG) of een verkorte versie.
+            type: string
+          vanuitVertrokkenOnbekendWaarheen:
+            $ref: '#/components/schemas/VertrokkenOnbekendWaarheen'
+          datumAanvangAdreshouding:
+            $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+          datumIngangGeldigheid:
+            $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+          datumInschrijvingInGemeente:
+            $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+          datumVestigingInNederland:
+            $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+          gemeenteVanInschrijving:
+            $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel"
+          landVanwaarIngeschreven:
+            $ref: 'land.yaml#/components/schemas/AbstractLand'
+          inOnderzoek:
+            $ref: "#/components/schemas/VerblijfplaatsInOnderzoek"
+    Locatie:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaats'
+        - type: object
+          properties:
+            functieAdres:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            locatiebeschrijving:
+              $ref: '#/components/schemas/Locatiebeschrijving'
+            woonplaats:
+              $ref: '#/components/schemas/Woonplaats'
+            inOnderzoek:
+              $ref: '#/components/schemas/LocatieInOnderzoek'
+    VerblijfplaatsOnbekend:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaats'
+        - type: object
+          properties:
```

Comment:
bij verblijfplaatsOnbekend moet ook inOnderzoek voorkomen, met daarin land (verblijfplaatsOnbekend is afgeleid van land).

Reply:
toegevoegd